src/ontology/uberon-edit.obo

```diff
@@ -214663,10 +214663,13 @@ property_value: provenance_notes "This class was sourced from an external ontolo
 
 [Term]
 id: UBERON:3011045
-name: gasserian ganglion
+name: obsolete gasserian ganglion
 synonym: "gasserian ganglia" RELATED [UBERON:3011045]
 is_a: UBERON:0000045 ! ganglion
 property_value: provenance_notes "This class was sourced from an external ontology (amphibian_anatomy). Its definitions, naming conventions and relationships may need to be checked for compatibility with uberon" xsd:string {source="http://purl.obolibrary.org/obo/aao.owl"}
+comment: Obsoleted as gasserian ganglion is an exact synonym to trigeminal ganglion per Wikipedia
```

Comment:
Wikipedia is not a great source for disambiguation terms - Such decisions should be made by an anatomist! Did you get any expert opinion on this?

Reply:
I consulted a biologist (not anatomist) and google. My understanding is what I propose with this PR. How else could this be resolved? The other term (UBERON:0001675) looks older, more developed and correct and without scary provenance notes.

The third alternative is to not do anything but this leaves the polysemy problem untouched.

I will check again with the biologist, there is nothing else I can do. Sorry...

Reply:
ok we double checked with the biologist. Here is another link that confirms it: The semilunar sensory ganglion (also known as the trigeminal ganglion or Gasserian ganglion) is a...

So I am pushing my changed PR.

Thanks a lot for reviewing and the hand holding in general.

Reply:
Looks good to me!